tox.ini

```diff
@@ -14,31 +14,6 @@ skip_missing_interpreters = {tty:True:False}
 
 basepython = python3.8
 
-setenv =
-    PY_MODULE=klein
-
-    PYTHONPYCACHEPREFIX={envtmpdir}/pycache
-
-
-##
-# Default environment: unit tests
-##
-
-[testenv]
-
-description = run tests
-
-basepython =
-    py27: python2.7
-    py35: python3.5
-    py36: python3.6
-    py37: python3.7
-    py38: python3.8
-    py39: python3.9
-
-    pypy2: pypy
-    pypy3: pypy3
-
```

Comment:
The diff output here is a bit counter-intuitive.  What's going on is that I moved the `deps=` config from the `[testenv]` up to `[default]` so that the same dependencies can be used by both `[testenv]` and `[mypy]` by referencing it from `[default]`.